source/MulensModel/binarylenswithshear.py

```diff
@@ -0,0 +1,617 @@
+import warnings
+import numpy as np
+from math import sqrt
+
+from MulensModel.binarylens import BinaryLens
+from MulensModel.binarylensimports import (_vbbl_wrapped,
+                                           _vbbl_binary_mag_0, _vbbl_SG12_9)
+import MulensModel as mm
+
+
+class BinaryLensWithShear(BinaryLens):
+    """
+    The binary lens equation - its solutions, images, parities,
+    magnifications, etc.
+
+    The binary lens equation is a 5th order complex polynomial
+    or a 9th order complex polynomial if including external shear.
+
+    Attributes :
+        mass_1: *float*
```

Comment:
If mass_1 and mass_2 are fractions of the TOTAL mass, I suggest using epsilon_1 and epsilon_2 instead. That way, it is clear that they are not true masses. Also, epsilon is the symbol commonly used from mass fraction in microlensing.

Reply:
It follows what we implemented for BinaryLens, so we will leave it as is for consistency.